plugins/linux-capture/pipewire.c

```diff
@@ -450,6 +452,7 @@ static uint32_t build_format_params(obs_pipewire_data *obs_pw,
 			obs_pw->format_info.array[i].modifiers.array,
 			obs_pw->format_info.array[i].modifiers.num);
 	}
+build_shm:
```

Comment:
Nitpick: please add a newline above `build_shm`:

```suggestion

build_shm:
```